### PR TITLE
Move configuration in to config file

### PR DESCRIPTION
**Pull request by kvsikand**
uses config reader, removes many of the cmd-line flags.

This code has been tested on the Jackal., and everything seems to be in order.

### CODE REVIEW

**Comment by kvsikand** [comment]
This latest travis-CI failure is because the default navigation map (UT_Campus) does not yet havea v2 map in the repo. Once this lands https://github.com/ut-amrl/amrl_maps/pull/23 , This build should pass

File: src/navigation/navigation.cc [hunk 1]
```diff
@@ -820,20 +799,21 @@ PathOption GetBestOption(const vector<PathOption>& options,
...
       o.dist_to_goal = FLT_MAX;
+      o.clearance = 0;
     }
-    const float omega_next = robot_vel_.x() * o.curvature;
-    if (omega_next < robot_omega_ - max_domega ||
-        omega_next > robot_omega_ + max_domega) {
-      o.free_path_length = 0;
-      o.dist_to_goal = FLT_MAX;
-    }
+    // const float omega_next = robot_vel_.x() * o.curvature;
```
Comment (joydeep-b) [inline src/navigation/navigation.cc]: This disables angular velocity dynamic constratints checking, it should not be commented out.

**Review by joydeep-b — CHANGES_REQUESTED** [review]

File: src/navigation/navigation.cc (hunk 1 above)
Comment (kvsikand) [reply src/navigation/navigation.cc]: Ok, will re-enable. I think this was just lifted from the existing diff

**Review by joydeep-b — APPROVED** [review]

**Comment by joydeep-b** [comment]
Should be tested before merging.

**Comment by kvsikand** [comment]
Was tested on the Jackal earlier today. Will go ahead and merge

**Pull request merged**